docs/source/tutorials/token_classification.ipynb

```diff
@@ -173,51 +178,56 @@
    "source": [
     "<details><summary>Below is the code used for reading the files.</summary>\n",
     "\n",
-    "    filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
-    "    given_entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC']\n",
-    "    merged_entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
-    "    entity_map = {entity: i for i, entity in enumerate(given_entities)} \n",
+    "```ipython3\n",
```

Comment:
Regarding the pulldown structure, what do you think about: merging all the code that appears in the pulldown into a single hidden code cell (below the pulldown). It would be a larger cell, but I think a cleaner approach (eg. would solve our current issue of forgetting to hide individual cells when there are many of them). If you make this change, I'd define the function at the top of this single cell and then have the executed code be all below the function definition rather than having the function definition appear in the middle of the code cell.

Reply:
IIRC, the function needs to be in the "middle" because it reference variables in "global" scope.